From ffd8d674a33ba4b6710fd51b3559deda52b0d502 Mon Sep 17 00:00:00 2001
From: Yuanjun Huang <yuanjun.huang@intel.com>
Date: Fri, 8 Jun 2018 15:01:50 +0800
Subject: [PATCH] BoardCongfig: Add option to indicate video driver readiness.

HWComposer will check this flag to disable va renderer if
va driver is not ready.

Change-Id: Ice9e791cdc5a55f237a499639e1122a199517f96
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-64421
Signed-off-by: Yuanjun Huang <yuanjun.huang@intel.com>
---
 cel_apl/BoardConfig.mk | 1 +
 celadon/BoardConfig.mk | 1 +
 2 files changed, 2 insertions(+)

diff --git a/cel_apl/BoardConfig.mk b/cel_apl/BoardConfig.mk
index de3e89b..ce6c0fe 100644
--- a/cel_apl/BoardConfig.mk
+++ b/cel_apl/BoardConfig.mk
@@ -25,6 +25,7 @@ USE_OPENGL_RENDERER := true
 NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 USE_INTEL_UFO_DRIVER := false
 INTEL_VA := true
+HWC_DISABLE_VA_DRIVER := true
 BOARD_GRAPHIC_IS_GEN := true
 BOARD_GPU_DRIVERS := i965
 BOARD_USE_MESA := true
diff --git a/celadon/BoardConfig.mk b/celadon/BoardConfig.mk
index b71408b..96ff5c2 100644
--- a/celadon/BoardConfig.mk
+++ b/celadon/BoardConfig.mk
@@ -25,6 +25,7 @@ USE_OPENGL_RENDERER := true
 NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 USE_INTEL_UFO_DRIVER := false
 INTEL_VA := true
+HWC_DISABLE_VA_DRIVER := true
 BOARD_GRAPHIC_IS_GEN := true
 BOARD_GPU_DRIVERS := i965
 BOARD_USE_MESA := true
-- 
1.9.1

